src/client/pages/Section/DataTable/Table/Table.tsx

```diff
@@ -56,8 +64,18 @@ const Table: React.FC<Props> = (props) => {
           {canClearData && <ButtonTableClear disabled={disabled} sectionName={sectionName} table={table} />}
         </div>
 
-        <DataGrid className="table-grid" gridTemplateColumns={`minmax(auto, 400px) repeat(${headers.length},1fr)`}>
-          <GridHead assessmentName={assessmentName} data={data} headers={headers} table={table} />
+        <DataGrid
+          className="table-grid"
+          gridTemplateColumns={`minmax(auto, 400px) repeat(${headers.length},1fr)`}
+          withActions={withActions}
+        >
+          <GridHead
+            assessmentName={assessmentName}
+            data={data}
+            headers={headers}
+            table={table}
+            withActions={withActions}
```

Comment:
GridHead has a loop and it renders the empty div for each row, so that wouldn't work. 
However, in a coming PR I will get rid of `GridRow`, so in that same PR I'll remove `GridHead` and render directly in the main Table component.